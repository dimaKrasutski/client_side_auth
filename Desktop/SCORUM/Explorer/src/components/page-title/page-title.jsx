import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';

export const PageTitle = ({ title, secondaryTitle }) => (
  <h1 className="mb-6">
    <FormattedMessage id={title} />
    {
      secondaryTitle &&
      <span className="text-gray-dark-secondary ml-2">
        <FormattedMessage id={secondaryTitle} />
      </span>
    }
  </h1>
);

PageTitle.propTypes = {
  title: PropTypes.string.isRequired,
  secondaryTitle: PropTypes.string,
};

PageTitle.defaultProps = {
  secondaryTitle: '',
};
