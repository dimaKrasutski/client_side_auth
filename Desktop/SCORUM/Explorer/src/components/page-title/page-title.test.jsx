import React from 'react';
import { PageTitle } from './page-title';

describe('<PageTitle />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PageTitle title="test" secondaryTitle="sec" />);
    expect(wrapper).toMatchSnapshot();
  });
});
