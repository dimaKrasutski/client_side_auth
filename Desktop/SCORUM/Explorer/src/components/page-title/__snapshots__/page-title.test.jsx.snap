// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> should match snapshot 1`] = `
<h1
  className="mb-6"
>
  <FormattedMessage
    id="test"
    values={Object {}}
  />
  <span
    className="text-gray-dark-secondary ml-2"
  >
    <FormattedMessage
      id="sec"
      values={Object {}}
    />
  </span>
</h1>
`;
